src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -223,14 +224,14 @@ values. The example that follows is typical of such classes.
 for this example app.
 
 This example is based on the APIs and tools provided by the
-[intl](https://pub.dartlang.org/packages/intl) package. [An alternative class
+[intl]({{site.pub}}/packages/intl) package. [An alternative class
 for the app's localized resources](#alternative-class) describes
 [an example](https://github.com/flutter/website/tree/master/examples/internationalization/minimal)
 that doesn't depend on the intl package.
 
 The DemoLocalizations class contains the app's strings (just one for the example)
 translated into the locales that the app supports. It uses the `initializeMessages()`
-function generated by Dart's [intl package](https://pub.dartlang.org/packages/intl)
+function generated by Dart's [intl package]({{site.pub}}/packages/intl)
 to load the translated strings, and
 [`Intl.message()`](https://www.dartdocs.org/documentation/intl/0.15.1/intl/Intl/message.html)
```

Comment:
GLOBAL: fix links to www.dartdocs.org to be to {{site.pub-api}} and remove the version number.

I think this link should be to `{{site.pub-api}}/intl/latest/intl/Intl/message.html`.

I didn't see any occurrences of {{site.pub}}/documentation, but you might want to check for that, too.

Maybe do a global search for `http://` and `https://` to catch any other links that we might want to macrofy or otherwise update.